src/python/platforms/android/adb.py

```diff
@@ -424,6 +442,25 @@ def reset_device_connection():
 
   return True
 
+def get_gce_device_ip():
+  try:
+    return socket.gethostbyname('cuttlefish')
+  except socket.gaierror as _:
+    logs.log_warn('Failed to get cuttlefish ip address, use localhost instead')
+    return '127.0.0.1'
+
+
+def set_gce_device_serial():
+  device_serial = '%s:6520' % get_gce_device_ip()
+  environment.set_value('ANDROID_SERIAL', device_serial)
+
+
+def get_gce_address():
+  ip = get_gce_device_ip()
+  if ip == '127.0.0.1':
+    return ''
+  return 'vsoc-01@%s' % ip
```

Comment:
where does vsoc-01 come from? perhaps put it in a constant somewhere in this file.

Reply:
Done